src/test/scala/apps/cameraPipeCheck.scala

```diff
@@ -215,10 +259,10 @@ int main(int argc, char** argv) {
   test("color correction passes checks") {
     val typed = printTime(infer(color_correct))
     println(s"color correction: ${typed.t}")
-    val lower: Strategy[Rise] = LCNF `;` CNF `;`
-      repeatNTimes(2, oncetd(lowering.mapSeq)) `;`
-      oncetd(lowering.mapSeqUnroll)
-    val lowered = printTime(lower(typed).get)
+    // val lower: Strategy[Rise] = LCNF `;` CNF `;`
+    //   repeatNTimes(2, oncetd(lowering.mapSeq)) `;`
+    //   oncetd(lowering.mapSeqUnroll)
+    val lowered = printTime(naiveLowering(typed).get)
```

Comment:
Is this successfully runing without the unrolling? Maybe because of the automatic unrolling we have in the codegen?